### PR TITLE
new doc: reproducing CI env locally

### PR DESCRIPTION
**Pull request by TomasTomecek**
Since we're changing how SRPMs are being done, I wanted to provide clear docs how one can reproduce the Copr env locally.

### CODE REVIEW

File: content/docs/reproduce-locally.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+```
+[root@f26be2947a15 /]# dnf install -y copr-rpmbuild
+```
+
+Now let's rerun locally a build for build-id 3580313:
+```
+[root@f26be2947a15 /]# /usr/bin/copr-rpmbuild --verbose --drop-resultdir --srpm --build-id 3580313
+```
+
+I hope it's clear that in order for this to work, your changes need to be
```
Comment (jpopelka) [inline content/docs/reproduce-locally.md:47]:
```suggestion
In order for this to work, your changes need to be
```

**Review by jpopelka — APPROVED** [review]

File: content/docs/reproduce-locally.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+"configuration.md#srpm_build_deps" >}}) is set in your packit config), this
+section describes how you can reproduce the build procedure locally.
+
+Thankfully, Copr is very transparent how it performs builds. When you open a
+build log (e.g. builder-live.log.gz), you'll see a command at the top which
+Copr invokes to perform a build. Let's do that in a container.
+
+```
+$ sudo podman run --privileged -ti fedora:36 bash
+```
```
Comment (praiskup) [inline content/docs/reproduce-locally.md:36]: Reading this once more ... are you sure `sudo` is actually needed?

Comment (TomasTomecek) [reply content/docs/reproduce-locally.md:36]: actually yes, I explained it below

if you can give me tips how to run mock in a container, I'd love to improve this line and drop as many privileges as possible

Comment (TomasTomecek) [reply content/docs/reproduce-locally.md:36]: Now I understand and you're correct, indeed sudo is not needed; thank you!

https://github.com/packit/packit.dev/pull/408

**Pull request merged**